playwright-e2e/tests/singular/contact-us/contact-us-page-visual.spec.ts

```diff
@@ -1,17 +1,18 @@
-import { expect } from '@playwright/test';
+import {expect, Locator, Page} from '@playwright/test';
 import { test } from 'fixtures/singular-fixture';
 
 /**
  * Functional tests for the Home page
  */
 test.describe('Home page', () => {
+
   test('Our teams @visual @singular', async ({ page, homePage }) => {
     await homePage.gotoURLPath('/about');
     const headerText = page.locator('.our-team h1');
 
-    expect(await headerText.screenshot()).toMatchSnapshot('our-team-text.png');
-    expect(await page.locator('.our-team .cards').screenshot({ mask: [page.locator('img')] })).toMatchSnapshot(
-      'our-team-cards.png'
+    await expect(headerText).toHaveScreenshot('our-team-text.png');
```

Comment:
Insert `await expect(headerText).toHaveText('Our Team');` at line #12. Then don't have to update screenshot `our-team-text-chromium-darwin.png`. Restore that screenshot.